src/wallet/test/wallet_tests.cpp

```diff
@@ -470,6 +470,59 @@ BOOST_FIXTURE_TEST_CASE(wallet_disableprivkeys, TestChain100Setup)
     BOOST_CHECK(!wallet->GetKeyFromPool(pubkey, false));
 }
 
+// This tests the possible conflict between maxtxfee and mintxfee/paytxfee
+BOOST_FIXTURE_TEST_CASE(wallet_maxfee, TestChain100Setup)
+{
+    // Mine a block, setup the wallet and scan to get some balance
+    CreateAndProcessBlock({}, GetScriptForRawPubKey(coinbaseKey.GetPubKey()));
+    auto chain = interfaces::MakeChain();
+    std::shared_ptr<CWallet> wallet = std::make_shared<CWallet>(chain.get(), WalletLocation(), WalletDatabase::CreateMock());
+    bool firstRun;
+    wallet->LoadWallet(firstRun);
+    AddKey(*wallet, coinbaseKey);
+    WalletRescanReserver reserver(wallet.get());
+    reserver.reserve();
+    CWallet::ScanResult result = wallet->ScanForWalletTransactions(::ChainActive().Genesis()->GetBlockHash(), {} /* stop_block */, reserver, false /* update */);
+
+    // Create a transaction with a lot of outputs so it's big enough to trigger the error at
+    // the configured paytxfee and mintxfee levels.
+    std::vector<CRecipient> recips = {};
+    recips.resize(8000,CRecipient{GetScriptForRawPubKey({}), 5000, false /* subtract fee */});
```

Comment:
nit, mi
missing space after `,` in a couple of places.

Reply:
Don't forget to install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script to preempt whitespace nitpicking.

Reply:
Thanks! Ran it, squashed & pushed.